packages/mds-rpc-common/server/index.ts

```diff
@@ -95,13 +96,18 @@ export const RpcServer = <S>(
   let server: Nullable<http.Server> = null
   let repl: Nullable<net.Server> = null
 
+  const httpServer = (application: Express, port: number): http.Server => {
+    const { customize = s => s } = options
```

Comment:
Could you add a comment explaining what these options do?

Reply:
Fixed in [ec0db83](https://github.com/lacuna-tech/mds-core/pull/813/commits/ec0db833a809be6137c8a4fdf5840a98d7b5d6de)